### DISCUSSION TITLE
Does it make sense to run MA seperate from HA ?

### DISCUSSION THREAD

**Discussion by nejckikelj** [music-assistant/support#1644] [Q&A] [answered]
Hello,

I'm new to MA and I'm thinking weather should I install MA as HA addon or is it better to run in a separate LXC with docker on my Proxmox hypervisor?

My goal is to keep things separate, so if something goes sideways, I still have control of the rest, that's why I run zigbe2mqtt in a seperate LXC, HA in a separate VM, ...

Are there any benefits in installing it as HA addon?

**Comment by OzGav** [comment, marked as answer]
Firstly, the addon is secured via the HA auth system as it is accessed via ingress. 
Secondly, the update process is easier as it is handled by the HA supervisor.
